resources/io/jenkins/infra/docker/Makefile

```diff
@@ -9,11 +9,12 @@ endif
 
 IMAGE_NAME ?= helloworld
 IMAGE_DEPLOY_NAME ?= "$(IMAGE_NAME)"
-IMAGE_PLATFORM ?= linux/amd64
+BUILD_TARGETPLATFORM ?= linux/amd64
```

Comment:
Ok for me but you have to write in PR title and description that it is a breaking change

Reply:
it's perfect, I did add a ':' and a warning icon to the makefile change in the body, is that good enough ? (thanks for the link for the conventionalcommits)